### PR TITLE
Insights Node.js SDK based on code generated from Swagger

### PR DESCRIPTION
**Pull request by gucalder**

### CODE REVIEW

File: lib/services/insights/README.md [hunk 1]
```diff
@@ -23,13 +23,8 @@ npm install azure-arm-insights
...
 
 ```javascript
-var common               = require("azure-common"),
-    insightsClientLib    = require("azure-arm-insights");
-
-var insightsManagementClient = insightsClientLib.createInsightsManagementClient(new common.TokenCloudCredentials({
-  subscriptionId: "<your subscription id>",
-  token: "<your token here>"
-}));
+var common               = require("ms-rest-azure"),
```
Comment (amarzavery) [inline lib/services/insights/README.md]: `var msrestAzure`. Please take a look at the [readme of storageManagement](https://github.com/Azure/azure-sdk-for-node/tree/master/lib/services/storageManagement2#how-to-use) as an example and model things accordingly for insights.

File: lib/services/insightsManagement/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+```bash
+npm install azure-arm-insights
+```
+
+## How to Use
+
+### Authentication
+
+ - TODO: Describe tokens
+
```
Comment (amarzavery) [inline lib/services/insightsManagement/README.md]: Pleae take a look at the readme of storage management (https://github.com/Azure/azure-sdk-for-node/tree/master/lib/services/storageManagement2#how-to-use) and make changes to isnisghts management accordingly.

File: lib/services/insights/package.json [hunk 3]
```diff
@@ -22,10 +22,10 @@
     "node",
     "azure"
   ],
-  "main": "./lib/insights.js",
+  "main": "./lib/insightsClient.js",
   "license": "Apache-2.0",
```
Comment (amarzavery) [inline lib/services/insights/package.json]: "license": "MIT"

File: lib/services/insightsManagement/package.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  ],
+  "keywords": [
+    "node",
+    "azure"
+  ],
+  "main": "./lib/insightsManagementClient.js",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "ms-rest-azure": "^1.15.0",
+    "moment": "^2.14.1"
```
Comment (amarzavery) [inline lib/services/insightsManagement/package.json]: "ms-rest": "^1.15.0"

File: package.json [hunk 5]
```diff
@@ -30,6 +30,7 @@
     "azure-common": "0.9.18",
     "azure-gallery": "2.0.0-pre.20",
     "azure-graph": "1.1.0",
+    "azure-insights": "0.14.4-preview",
```
Comment (amarzavery) [inline package.json]: In package.json of insights you have"0.14.0-preview"

File: package.json [hunk 6]
```diff
@@ -56,7 +57,7 @@
     "azure-arm-eventhub": "0.1.0",
     "azure-arm-hdinsight": "0.2.1",
     "azure-arm-hdinsight-jobs": "0.1.1",
-    "azure-arm-insights": "0.11.4",
+    "azure-arm-insights": "0.14.4-preview",
```
Comment (amarzavery) [inline package.json]: In package.json of insightsManagement you have"0.14.0-preview"

File: lib/services/insightsManagement/package.json [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+  "tags": [
+    "azure",
+    "sdk"
+  ],
+  "keywords": [
+    "node",
+    "azure"
+  ],
+  "main": "./lib/insightsManagementClient.js",
+  "license": "Apache-2.0",
```
Comment (amarzavery) [inline lib/services/insightsManagement/package.json]: "license": "MIT"

File: lib/services/insightsManagement/package.json [hunk 8]
```diff
@@ -0,0 +1,42 @@
+{
```
Comment (amarzavery) [inline lib/services/insightsManagement/package.json]: Please update the content of License.txt in this folder to that of MIT license. You can copy this content from [here](https://raw.githubusercontent.com/Azure/azure-sdk-for-node/master/lib/services/storageManagement2/LICENSE.txt)

File: lib/services/insights/package.json [hunk 9]
```diff
@@ -22,10 +22,10 @@
...
   "dependencies": {
-    "azure-common": "^0.9.17",
+    "ms-rest-azure": "^1.15.0",
```
Comment (amarzavery) [inline lib/services/insights/package.json]: Also add the generic runtime,
`"ms-rest": "^1.15.0"`

File: lib/services/insights/package.json [hunk 10]
```diff
@@ -1,5 +1,5 @@
 {
```
Comment (amarzavery) [inline lib/services/insights/package.json]: Please update the content of License.txt in this folder(lib/services/insights) to that of MIT license. You can copy this content from [here](https://raw.githubusercontent.com/Azure/azure-sdk-for-node/master/lib/services/storageManagement2/LICENSE.txt)

File: lib/services/insights/package.json (hunk 3 above)
Comment (gucalder) [inline lib/services/insights/package.json]: Fixed

File: lib/services/insights/package.json (hunk 9 above)
Comment (gucalder) [inline lib/services/insights/package.json]: Added.

File: lib/services/insightsManagement/package.json (hunk 4 above)
Comment (gucalder) [inline lib/services/insightsManagement/package.json]: Added

File: lib/services/insightsManagement/package.json (hunk 7 above)
Comment (gucalder) [inline lib/services/insightsManagement/package.json]: Fixed.

File: package.json (hunk 6 above)
Comment (gucalder) [inline package.json]: A typo... fixed, I am using 0.14.0-preview everywhere

File: package.json (hunk 5 above)
Comment (gucalder) [inline package.json]: Typo.. fixed.

File: lib/services/insightsManagement/package.json (hunk 8 above)
Comment (gucalder) [inline lib/services/insightsManagement/package.json]: Done.

File: lib/services/insights/package.json (hunk 10 above)
Comment (gucalder) [inline lib/services/insights/package.json]: Done

File: lib/services/insightsManagement/README.md (hunk 2 above)
Comment (gucalder) [inline lib/services/insightsManagement/README.md]: Done

File: lib/services/insights/README.md (hunk 1 above)
Comment (gucalder) [inline lib/services/insights/README.md]: Done

**Pull request merged**